File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:

        temp = 0 
        count = 0 

        for i in nums:
            if i == 1: 
                temp = temp + 1
            else: #if zero 
                if temp > count:
                    count = temp 
                temp = 0 
        if temp > count:
            count = temp 

        return count

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size() * 2;
        vector<int> ans(n);
        for (int i = 0; i < n; i++) {
            ans[i] = nums[i % (n/2)];
        }

        return ans;
    }
};